ui/src/components/Drawer/components/SessionDetail/index.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { storeToRefs } from 'pinia';
+
+import { useConnectionStore } from '@/store/modules/useConnection';
+
+const { t } = useI18n();
+
+const connectionStore = useConnectionStore();
+
+const { user, account, asset, protocol, date_start, date_end } = storeToRefs(connectionStore);
+
+const descriptions = computed(() => {
+  return [
+    {
+      label: t('User'),
+      value: user?.value || '-',
+    },
+    {
+      label: t('AccountMessage'),
+      value: account?.value || '-',
+    },
+    {
+      label: t('Asset'),
+      value: asset?.value || '-',
+    },
+    {
+      label: t('Protocol'),
+      value: protocol?.value || '-',
+    },
+    {
+      label: t('DateStart'),
+      value: date_start?.value || '-',
+    },
+    {
+      label: t('DateEnd'),
+      value: date_end?.value || '-',
+    },
+  ];
+});
+</script>
+
+<template>
+  <n-descriptions bordered label-placement="left" :column="1">
+    <n-descriptions-item v-for="item in descriptions" :key="item.label" :label="item.label">
+      {{ item.value }}
+    </n-descriptions-item>
+  </n-descriptions>
+</template>
```

Comment:
The code appears to be perfectly fine. There are no issues, regular expressions were properly closed, proper spacing has been used in all places it is required; there's not much you can suggest without specific context on what "irregularity", "potential issue", or anything else means.

If you want recommendations for improvements (such as optimizing performance with Vue.js), additional functionality additions etc., we'd need more details about the current state of application that uses this component so these kinds of ideas could help inform future developments. But based primarily just on existing text, there isn't enough detail provided here to make any suggestions that aren't covered within the guidelines given.